src/test/scala/chiselTests/BoringUtilsSpec.scala

```diff
@@ -457,4 +457,66 @@ class BoringUtilsSpec extends ChiselFlatSpec with ChiselRunners with Utils with
       "foo.b_bore <= read(bar.b_bore)"
     )()
   }
+
+  "Downwards writable tap from grandparent to grandchild" should "work" in {
+    class Bar extends RawModule {
+      val internalWire = Wire(Bool())
+    }
+    class Foo extends RawModule {
+      val bar = Module(new Bar)
+    }
+    class Top extends RawModule {
+      val foo = Module(new Foo)
+      val out = IO(Bool())
+      out := probe.read(BoringUtils.rwTap(foo.bar.internalWire))
```

Comment:
Kind of a nit, but could you do both a `probe.read` and a `probe.force`? Both should work.